TrainingTask/MyCar.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrainingTask
+{
+    public class MyCar : Drawer
+    {
+        private TrackField trackField = new TrackField();
```

Comment:
Default state should be set in a constructor.

Reply:
-1 to source code quality.

Reply:
Fixed.